packages/ui/components/pages/Fuse/FusePoolsPage/FusePoolList.tsx

```diff
@@ -65,12 +69,16 @@ const FusePoolList = () => {
     setCurrentPage(nextPage);
   };
 
-  if (error) {
+  useEffect(() => {
+    setErr(error as Err);
+  }, [error]);
+
+  if (err && err.code !== 'NETWORK_ERROR') {
     return (
       <AlertHero
         status="warning"
         variant="subtle"
-        title="Unexpected Error"
+        title={err.reason ? err.reason : 'Unexpected Error'}
```

Comment:
why not use `error` directly? It should cause a rerender the same way as setting a state does. 🤔